taichi/runtime/runtime.cpp

```diff
@@ -207,6 +215,16 @@ f64 pow_f64(f64 a, f64 b) {
   return std::pow(a, b);
 }
 
+#if ARCH_cuda
```

Comment:
For `pow_i32` and `nv_pow_i32` we may want to use exponentiation by squaring, for accuracy reasons. `std::pow` might nit have enough accuracy here.

Reply:
...and then `nv_pow_x` and `pow_x` can get merged since they are the same.